airflow/api_fastapi/core_api/datamodels/connections.py

```diff
@@ -90,8 +91,74 @@ class ConnectionBody(BaseModel):
     extra: str | None = Field(default=None)
 
 
-class ConnectionBulkBody(BaseModel):
-    """Connections Serializer for requests body."""
+class ConnectionBulkCreateAction(BaseModel):
+    """Bulk Create Variable serializer for request bodies."""
+
+    action: Literal["create"] = "create"
```

Comment:
Maybe we should switch to an enum for the `action`. To avoid hardcoding everywhere `create` `delete` ... strings.

Reply:
Updated as an Enum and applied the changes to `variables` too.